tests/e2e/run-scanner-v4-install.bats

```diff
@@ -596,6 +596,95 @@ EOT
     _end
 }
 
+@test "Fresh installation of HEAD Helm charts in the same namespace and toggling Scanner V4" {
+    local password_setting=$(cat <<EOT
+central:
+  adminPassword:
+    value: "$ROX_ADMIN_PASSWORD"
+EOT
+    )
+    local secured_cluster_name="$(get_cluster_name)"
+    namespace="stackrox"
+
+    ######################
+    _begin "deploying-head-central"
+    info "Deploying central-services using HEAD chart"
+    deploy_central_with_helm "$namespace" "$MAIN_IMAGE_TAG" "" \
+        -f <(echo "$password_setting")
+    local central_endpoint="$(get_central_endpoint "$namespace")"
+
+    ######################
+    _begin "deploying-head-sensor"
+    info "Deploying secured-cluster-services using HEAD chart"
+    deploy_sensor_with_helm "$namespace" "$namespace" \
+        "$MAIN_IMAGE_TAG" "" \
+        "$secured_cluster_name" "$ROX_ADMIN_PASSWORD" "$central_endpoint"
+
+    ######################
+    _step "verifying-central-scanners-deployed"
+    info "Verifying that scanners are deployed"
+    verify_scannerV2_deployed "$namespace"
+    verify_scannerV4_deployed "$namespace"
+    info "Verifying that scanner V4 is enabled for sensor"
+    verify_deployment_scannerV4_env_var_set "$namespace" "central"
+
+    ######################
+    _step "verifying-sensor-scanners-enabled"
+    info "Verifying that scanner V4 is enabled for sensor"
+    run verify_deployment_scannerV4_env_var_set "$namespace" "sensor"
```

Comment:
Is there a reason why this call is prefixed with `run` but the one for central above is not?